src/components/timeline/LayerDiscrete.svelte

```diff
@@ -404,76 +462,108 @@
   }
 
   function drawGroupedMode() {
+    // must clear the canvas before a redraw! otherwise the old ungrouped version can linger around and we draw over that!
```

Comment:
This already occurs at the beginning of the `draw` call which then calls this function so should not be necessary.

Reply:
That's what I thought, but for some reason that clear didn't actually clear the canvas successfully, despite the code definitely being reached. This was addressed in a commit a while back, but removing this call causes a ton of overlap between successive redraws!

Reply:
Hm I am not seeing this behavior when I remove the clear call.

Reply:
It must've been corrected by something else. It is now removed in [remove unnecessary clearing in `drawGroupedMode`](https://github.com/NASA-AMMOS/aerie-ui/commit/dea22eebf26d2c8b2cae12630072fdbc0511caac).